### DISCUSSION TITLE
ppc64le architecture support

### DISCUSSION THREAD

**Discussion by sebacea** [quarto-dev/quarto-cli#6249] [Q&A] [answered]
### Description

Hi, thanks for this useful environment. Do you know a way to use quarto in PowerPC?

I was not able to install from source

**Comment by mcanouil** [comment]
I don't even know what that is 😅
From what I gathered you should be able to use the deb file, did you try?

**Comment by sebacea** [comment]
PowerPC is [ppc64le architecture](https://en.wikipedia.org/wiki/Ppc64), it is what IBM Academic provides

It is not possible to use .deb from another architecture

**Comment by mcanouil** [comment]
FYI, there is a Docker image (200+ actually) with that arch which might be use to try to package Quarto.

https://hub.docker.com/r/ppc64le/ubuntu

Although, this needs:
- to have knowledge of this architecture,
- that all Quarto software dependencies exist for that architecture (not that sure).

Community help on this would be appreciated.

PS: I am not sure this arch is a "academics" arch, as in 10+ years in academia I never encountered it.

**Comment by sebacea** [comment]
I currently run a docker image with rstudio in the IBM servers (https://github.com/ppc64le/build-scripts/tree/master/r/rstudio/Dockerfiles) but quarto freezes the rstudio session somehow. Probably, because not all “dependencies exist for that architecture”. This is at the very origin of my question. 

and… Absolutely, this architecture is not at all an academic one. Is what the IBM Academics program provides only (probably my sentences was not correct en the previousbpost) I am also a 10+ years old academic and I agree with you. 

Btw I will be happy yo share my experience with the dockerfile with rstudio and quarto if you want

**Reply by mcanouil** [reply, marked as answer]
FYI, I created an issue for enhancement, but I feel it's going to be something on which community/users could help do via a PR.

- #6489
